cpp/dolfinx/mesh/utils.h

```diff
@@ -747,7 +747,7 @@ compute_incident_entities(const Topology& topology,
 /// 'nodes' will be included. See dolfinx::io::cells for examples of the
 /// Basix ordering.
 /// @param[in] element Coordinate element for the cells.
-/// @param[in] commg
+/// @param[in] commg Communicator for geometry
```

Comment:
This docstring says very little. We need to aim to be really clear. It is the communicator that the coordinate data is distributed across. Is it `MPI_COMM_NULL` for processes without geometry data?

Reply:
Not sure, I didn't add it.